query_postal_codes_heatmaps_api.py

```diff
@@ -0,0 +1,103 @@
+import random
+from collections import defaultdict
+import statistics
+
+from datetime import datetime, timedelta, date
+import matplotlib.pyplot as plt
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+
+
+def build_api_query(date: date, tile_ids: list) -> str:
```

Comment:
If you want you can take it one step further with `import typing as T` and then:
```suggestion
def build_api_query(date: date, tile_ids: T.Sequence[int]) -> str:
```

Reply:
I rather keep it simple :)